### PR TITLE
Add Readme file because it was missing

### PR DESCRIPTION
**Pull request by Felipeceballos12**
He contribuido agregando el archivo Readme al proyecto con una breve explicación sobre el proyecto.

### CODE REVIEW

**Comment by Felipeceballos12** [comment]
Muchas gracias 😊

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Lightning
+
+A free web application to practice your English speaking skills.
```
Comment (lmac-1) [inline Readme.md:3]: Luis, this readme is informative and great

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable, Efficient, Elegant, Maintainable

Comment (lmac-1) [reply Readme.md:3]: (Testing Sema)

In computer science, a linked list is a linear collection of data elements, in which linear order is not given by their physical placement in memory. Instead, each element points to the next. It is a data str...

📄 [Oleksii Trekhleb](https://github.com/trekhleb/javascript-algorithms/tree/master/src/data-structures/linked-list)

Comment (lmac-1) [reply Readme.md:3]: (Testing sema)

In computer science, a linked list is a linear collection of data elements, in which linear order is not given by their physical placement in memory. Instead, each element points to the next. It is a data structure consisting of a group of nodes which together represent a sequence. This structure allows for efficient insertion or removal of elements from any position in the sequence during iteration. A drawback of linked lists is that access time is linear (and difficult to pipeline). Faster access, such as random access, is not feasible.

📄 [Oleksii Trekhleb](https://github.com/trekhleb/javascript-algorithms/tree/master/src/data-structures/linked-list)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Efficient

Comment (lmac-1) [reply Readme.md:3]: Testing

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Reusable

Comment (lmac-1) [reply Readme.md:3]: (Testing Sema)

In computer science, a linked list is a linear collection of data elements, in which linear order is not given by their physical placement in memory. Instead, each element points to the next. It is a data structure consisting of a group of nodes which together represent a sequence. This structure allows for efficient insertion or removal of elements from any position in the sequence during iteration. A drawback of linked lists is that access time is linear (and difficult to pipeline). Faster access, such as random access, is not feasible.

📄 [Oleksii Trekhleb](https://github.com/trekhleb/javascript-algorithms/tree/master/src/data-structures/linked-list)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient

**Review by lmac-1 — COMMENTED** [review]
Luis, this is great

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable, Efficient

File: Readme.md (hunk 1 above)
Comment (lmac-1) [reply Readme.md:3]: This looks great!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by ovi-sema** [comment]
lgtm

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Pull request merged**